app/javascript/packages/document-capture/components/address-search.tsx

```diff
@@ -191,8 +194,9 @@ function AddressSearch({
   }, [isLoading]);
 
   useEffect(() => {
-    error && onError(error);
-  }, [error]);
+    addressError && onError(addressError);
+    uspsError && onError(uspsError);
```

Comment:
could we have both things error? If so, would we want to call both side effects? or just call the first?

Reply:
I wouldn’t expect both API calls to error.  If there’s an address search error, then there wouldn’t be an address to use for a call to the USPS API.